x-pack/plugins/apm/public/components/app/correlations/ml_latency_correlations.tsx

```diff
@@ -349,33 +349,29 @@ export function MlLatencyCorrelations({ onClose }: Props) {
         </>
       )}
       <EuiSpacer size="m" />
-      {overallHistogram !== undefined ? (
-        <>
-          <EuiTitle size="xxs">
-            <h4 data-test-subj="apmCorrelationsLatencyCorrelationsChartTitle">
-              {i18n.translate(
-                'xpack.apm.correlations.latencyCorrelations.chartTitle',
-                {
-                  defaultMessage:
-                    'Latency distribution for {name} (Log-Log Plot)',
-                  values: {
-                    name: transactionName ?? serviceName,
-                  },
-                }
-              )}
-            </h4>
-          </EuiTitle>
 
-          <CorrelationsChart
-            markerPercentile={DEFAULT_PERCENTILE_THRESHOLD}
-            markerValue={percentileThresholdValue ?? 0}
-            {...selectedHistogram}
-            overallHistogram={overallHistogram}
-          />
+      <EuiTitle size="xxs">
+        <h4 data-test-subj="apmCorrelationsLatencyCorrelationsChartTitle">
+          {i18n.translate(
+            'xpack.apm.correlations.latencyCorrelations.chartTitle',
+            {
+              defaultMessage: 'Latency distribution for {name} (Log-Log Plot)',
```

Comment:
Do we need to keep `Log-Log Plot` in this label? Makes it more obvious I guess? But if left in it should probably be sentence case `Log-log plot`

Reply:
Good point. No we just want this:

```suggestion
              defaultMessage: 'Latency distribution',
```